client/auth/signup.jsx

```diff
@@ -181,10 +179,10 @@ export default class Signup extends React.Component {
   }
 
   onLogInClick = () => {
-    const query = {
+    const search = queryString.stringify({
```

Comment:
Note to self: this is unnecessary since `this.props.location.search` is already string (oh and it's `search`, not `query` anymore).

Will fix when I come home from work.